source/includes/steps-create-index.yaml

```diff
@@ -27,18 +27,55 @@ content: |
 
    |compass-short| supports the following index options:
 
-   * `Build the index in the background <https://docs.mongodb.com/master/core/index-creation/index.html#background-construction>`_
+   .. list-table::
+      :header-rows: 1
+      :widths: 40 60 20
 
-   * :manual:`Create a unique index </core/index-unique>`
+      * - Option
+        - Description
+        - More Information
 
-   * :manual:`Create a TTL Index </core/index-ttl>`
+      * - Build index in the background
 
-   * :manual:`Create a partial index </core/index-partial/>`
+        - If checked, the MongoDB deployment remains available during
+          the index build operation.
 
-   * `Use custom collation <https://docs.mongodb.com/manual/indexes/index.html#indexes-and-collation>`_
+        - :manual:`Background Construction </core/index-creation/index.html#background-construction>`
 
-     For information on the Custom Collation settings, see :manual:`Collation Document </reference/collation/#collation-document>`
-     in the MongoDB manual.
+      * - Create unique index
+
+        - If checked, ensure that the indexed fields do not
+          store duplicate values.
+
+        - :manual:`Unique Indexes </core/index-unique>`
+
+      * - Create :abbr:`TTL (Time to Live)`
+
+        - If checked, automatically delete documents after a
+          specified number of seconds since the indexed field value.
+
+        - :manual:`TTL Indexes </core/index-ttl>`
+
+      * - Partial filter expression
```

Comment:
Are these options in the UI or did you name them? If possible, it would more consistent if these were all syntactically parallel. For example: "Use partial filter expression".

Reply:
These are all how the options are shown in the UI. I debated whether it would be better to just go with what is in the UI vs name them ourselves. Ultimately I went with what is in the UI, but I can talk with product and see if they are opening to making these options more parallel in the UI.

Reply:
I figured. And I think its right to go with whats in the UI, prevents confusion. But see if can you can maybe bend product's ear ;)